src/test/scala/mesosphere/marathon/api/SystemResourceTest.scala

```diff
@@ -101,7 +102,7 @@ class SystemResourceTest extends AkkaUnitTest with JerseyTest {
       response.getStatus shouldBe 200
       Option(response.getMetadata.getFirst("Content-Type")).value.toString should be("application/json")
 
-      val metricsJson = Json.parse(response.getEntity.asInstanceOf[String])
+      val metricsJson = Json.parse(response.getEntity.asInstanceOf[RestStreamingBody[_]].toString)
```

Comment:
```suggestion
      val metricsJson = Json.parse(response.getEntity.toString)
```
also works. What do you think?

Reply:
Here and everywhere else